venvUtils/venvCmd.bat

```diff
@@ -23,5 +23,4 @@ if ERRORLEVEL 1 goto :EOF
 echo call %*
 call %*
 echo Deactivating NVDA Python virtual environment
-call "%~dp0\..\.venv\scripts\deactivate.bat"

```

Comment:
What does `deactivate.bat` do? shouldn't those environment variables be cleaned up in some way?

Reply:
deactivate.bat sets particular environment variables back to their old values.
However, as we called setlocal before ensureAndActivate.bat, all the environment variables will go back to their original values once either endlocal is called, the script ends, or the script is aborted with control+c.

Reply:
Oh ok. Might be worth a comment to point that out. Otherwise this all looks good.